package.json

```diff
@@ -5,6 +5,7 @@
   "keywords"      : ["model", "view", "controller", "router", "server", "client", "browser"],
   "author"        : "Jeremy Ashkenas <jeremy@documentcloud.org>",
   "dependencies"  : {
+    "jquery"      : "^2.1.4",
```

Comment:
The two big issues are we don't know if the user wants `^1.x` or `^2.x` and we do not want to force the user to download jquery if they are using BB on the server

See #3421